kclvm/tools/src/fix/replace.rs

```diff
@@ -0,0 +1,271 @@
+//! A small module giving you a simple container that allows easy and cheap
```

Comment:
Pls note that adding the license file from the cargo code or put it into 3rdparty folders.